### ISSUE TITLE
v2 Bug on mobile devices

### ISSUE THREAD

**Issue by G-dogger** [marioloncarek/megamenu-js#14]
The v2 megamenu works fine on computers, but not on mobile devices; including Android and iPhone.

When you open the menu and try to scroll up or down through sub-menu choises, the menu closes. Just the slightest touch and it closes. Please help! Thanks! Example can be found here: http://www.hannabery.com/01homev2.html

**Comment by G-dogger** [comment]
I found the part of the code causing the problem, just don't know how to fix it. In the JS file at end: 

  $(window).resize(function () {
        if (getWindowWidth() < responsiveBreakpoint) {
            $('.menu > ul').removeClass('is-shown-on-mobile');
            $('.menu li').children('ul').hide();
            $('.dropdown-button-text').removeClass('is-active');
            $('.hamburger').removeClass('is-active');
        } else {
            $('.menu > ul > li > ul > li').children('ul').show();
        }
    });

If you remove this section, the menu will work fine on a phone, but then you can't use it properly on a computer when resizing browser with expanded menu - it breaks. Hope you can provide a fix!!!

**Comment by born2net** [comment]
yes I see the same issue, I really would like to use this menu but ya seems like a show stopper, as soon as you slide the menu closes up.

**Comment by G-dogger** [comment]
Mario, I installed v2 files, but it does not work properly on smart phones. The problem is caused by the window.resize function. Hoping you can provide a fix???????????

**Comment by G-dogger** [comment]
Maybe you need to check for document_height; as well as width for mobiles???

**Comment by leekangsan** [comment]
Yes, I have the same problem.  Hoping you can provide a fix???????????

**Comment by G-dogger** [comment]
I will give it a shot, thanks
